synology_dsm/api/surveillance_station/__init__.py

```diff
@@ -0,0 +1,133 @@
+        """Return camera live view path matching camera_id (video_format: mjpeg_http | multicast | mxpeg_http | rtsp_http | rtsp)."""
+        if video_format:
+            return getattr(self._cameras_by_id[camera_id].live_view, video_format)
+        return self._cameras_by_id[camera_id].live_view
+
+    def get_camera_image(self, camera_id):
+        """Return bytes of camera image for camera matching camera_id."""
+        return self._dsm.get(
+            self.CAMERA_API_KEY, "GetSnapshot", {"id": camera_id, "cameraId": camera_id}
+        )
+
+    def enable_camera(self, camera_id):
+        """Enable camera(s) - multiple ID or single ex 1 or 1,2,3."""
+        return self._dsm.get(self.CAMERA_API_KEY, "Enable", {"idList": camera_id})[
+            "success"
+        ]
+
+    def disable_camera(self, camera_id):
+        """Disable camera(s) - multiple ID or single ex 1 or 1,2,3."""
+        return self._dsm.get(self.CAMERA_API_KEY, "Disable", {"idList": camera_id})[
+            "success"
+        ]
+
+    # Snapshot
+    def capture_camera_image(self, camera_id, save=True):
+        """Capture a snapshot for camera matching camera_id."""
+        return self._dsm.get(
+            self.SNAPSHOT_API_KEY,
+            "TakeSnapshot",
+            {
+                "camId": camera_id,
+                "blSave": int(save),  # API requires an integer instead of a boolean
+            },
+        )
+
+    def download_snapshot(self, snapshot_id, snapshot_size):
+        """Download snapshot image binary for a givent snapshot_id (snapshot_size: SNAPSHOT_SIZE_ICON | SNAPSHOT_SIZE_FULL)."""
+        return self._dsm.get(
+            self.SNAPSHOT_API_KEY,
+            "LoadSnapshot",
+            {"id": snapshot_id, "imgSize": snapshot_size},
+        )
+
+    # Motion
+    def is_motion_detection_enabled(self, camera_id):
+        """Return motion setting matching camera_id."""
+        return self._cameras_by_id[camera_id].is_motion_detection_enabled
+
+    def enable_motion_detection(self, camera_id):
+        """Enable motion detection for camera matching camera_id."""
+        return self._dsm.get(
+            self.CAMERA_EVENT_API_KEY,
+            "MDParamSave",
+            {"camId": camera_id, "source": MOTION_DETECTION_BY_SURVEILLANCE},
+        )
+
+    def disable_motion_detection(self, camera_id):
+        """Disable motion detection for camera matching camera_id."""
+        return self._dsm.get(
+            self.CAMERA_EVENT_API_KEY,
+            "MDParamSave",
+            {"camId": camera_id, "source": MOTION_DETECTION_DISABLED},
+        )
+
+    # Home mode
+    def get_home_mode_status(self):
+        """Get the state of Home Mode"""
+        return self._dsm.get(self.HOME_MODE_API_KEY, "GetInfo")["data"]["on"]
+
+    def set_home_mode(self, state):
+        """Set the state of Home Mode (state: bool)"""
+        return self._dsm.get(self.HOME_MODE_API_KEY, "Switch", {"on": state})["success"]
```

Comment:
```suggestion
        return self._dsm.get(self.HOME_MODE_API_KEY, "Switch", {"on": str(state).lower()})["success"]
```
or something equivalent. It should be in lower case to make it work.